dll/win32/shimgvw/shimgvw.c

```diff
@@ -959,6 +977,58 @@ ImageView_OnSize(HWND hwnd, UINT state, INT cx, INT cy)
     }
 }
 
+static LRESULT CALLBACK
+ImageView_Delete(HWND hwnd)
+{
+    DPRINT1("ImageView_Delete: unimplemented.\n");
+    return 0;
+}
+
+static LRESULT CALLBACK
```

Comment:
```suggestion
static LRESULT
```
This is not callback.